docs/shared_parsers_catalog/sysconfig_network.rst

```diff
@@ -0,0 +1,4 @@
+.. automodule:: insights.parsers.sysconfig_network
+   :members:
+   :show-inheritance:
+
```

Comment:
Please remove this blank line if you insist to keep this file.

Reply:
As per latest organization of the code, this file is not required.